### PR TITLE
[release] merging pending into master

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin @sdroege

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,24 +7,45 @@ autobins = false
...
-glib-sys = "^0"
-gobject-sys = "^0"
-glib = "^0"
-gio = "^0"
-gdk = "^0"
-gdk-pixbuf = "^0"
-gtk = "^0"
-pango = "^0"
-cairo-rs = { version = "^0", features = ["png"] }
+futures-preview = { version = "0.3.0-alpha.15", optional = true}
```
Comment (EPashkin) [inline Cargo.toml]: This is wrong change

Comment (EPashkin) [reply Cargo.toml]: I meant: master need use packages, not gits

Comment (GuillaumeGomez) [reply Cargo.toml]: A surprising one for sure.

Comment (GuillaumeGomez) [reply Cargo.toml]: Don't worry, I got it. ;)

**Comment by GuillaumeGomez** [comment]
I'll merge it and send a PR for the specific dependencies change.

**Pull request merged**